spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Guest, type: :feature do 
+  let!(:comfy_palace) { Hotel.create!(name: "Comfy Palace", location: "Buena Vista") }
+
+  let!(:room_1) { comfy_palace.rooms.create!(rate: 100, suite: "Presidential") }
+  let!(:room_2) { comfy_palace.rooms.create!(rate: 200, suite: "Couples") }
+  let!(:room_3) { comfy_palace.rooms.create!(rate: 75, suite: "Economy") }
+
+  let!(:hannah) { Guest.create!(name: "Hannah", nights: 3) }
+
+  let!(:hannah_room_1) {GuestRoom.create!(guest_id: hannah.id, room_id: room_1.id) }
+  let!(:hannah_room_2) {GuestRoom.create!(guest_id: hannah.id, room_id: room_2.id) }
+
+  describe 'Guest show page' do 
+    before (:each) do 
+      visit "/guests/#{hannah.id}"
+    end
```

Comment:
Great job adding the `visit` to your `before :each`.

I'm always weary about using `let` with Rails, but as long as you understand the memoization piece behind it then I guess it's ok.